### DISCUSSION TITLE
Time error when upload mp4

### DISCUSSION THREAD

**Discussion by WangHaoyu0621** [immich-app/immich#13624] [Q&A]
### The bug

when I upload a video (shot by Nikon z30, GMT+0800, using website to upload), the displayed time is eight hours later than the actual shooting time.
eg. The video taken at 8:00(GMT+8) will become 16:00(GMT+8) in the app.

The time of video displayed on the computer correctly
No same bug when uploading photos

### The OS that Immich Server is running on

Unraid 6.9.2

### Version of Immich Server

1.118.1

### Version of Immich Mobile App

1.118.0

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
just default setting
only changed port
```

### Your .env content

```Shell
postage 16
```

### Reproduction steps

1. upload a video
2. the time recorded is wrong

**Comment by mmomjian** [comment]
You should set the time zone by specifying TZ in the env file, which you have not done. Please do this, refresh metadata for that video, and if it still doesn’t work you will need to post the entire info in #12650
